### PR TITLE
feat: add flag show-events

### PR DESCRIPTION
**Pull request by shubham-cmyk**
## Explanation
Add support to turn off events

Fixes https://github.com/kyverno/chainsaw/issues/835

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
- [x] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
- [ ] This is a bug fix and I have added unit tests that prove my fix is effective.

### CODE REVIEW

File: pkg/runner/collect/describe.go [hunk 1]
```diff
@@ -32,5 +33,8 @@ func Describe(collector *v1alpha1.Describe) (*v1alpha1.Command, error) {
 		namespace = "$NAMESPACE"
 	}
 	cmd.Args = append(cmd.Args, "-n", namespace)
+	if collector.ShowEvents != nil {
+		cmd.Args = append(cmd.Args, fmt.Sprintf("--show-events=%t", *collector.ShowEvents))
```
Comment (eddycharly) [inline pkg/runner/collect/describe.go:37]: `%t` is for bool ?

**Comment by eddycharly** [comment]
@shubham-cmyk a unit test would be nice.

File: pkg/runner/collect/describe.go (hunk 1 above)
Comment (shubham-cmyk) [reply pkg/runner/collect/describe.go:37]: %t	the word true or false

**Review by eddycharly — DISMISSED** [review]
Good job !

**Comment by shubham-cmyk** [comment]
need a codegen here

**Pull request merged**